services/ui-src/src/components/menus/Menu.tsx

```diff
@@ -20,13 +26,16 @@ export const Menu = ({ handleLogout }: Props) => {
       <Box role="group">
         <MenuButton
           as={Button}
-          rightIcon={<Icon icon="chevronDown" color="palette.white" />}
+          rightIcon={
```

Comment:
Request: Kinda unrelated but this would be a good time to change it. The menu list should not be getting a hover state background color change. We can just add `background: "none !important"` to the `_hover: {}` of `menuButton` down in sx for now.

Reply:
Done!